registry/repository.go

```diff
@@ -42,14 +42,14 @@ func (c *repositoryClient) ListSignatures(ctx context.Context, desc ocispec.Desc
 // FetchSignatureBlob returns signature envelope blob and descriptor given
 // signature manifest descriptor
 func (c *repositoryClient) FetchSignatureBlob(ctx context.Context, desc ocispec.Descriptor) ([]byte, ocispec.Descriptor, error) {
-	sigManifest, err := c.getSignatureManifest(ctx, desc)
+	signatureBlobs, err := c.getSignatureBlobDesc(ctx, desc)
 	if err != nil {
 		return nil, ocispec.Descriptor{}, err
 	}
-	if len(sigManifest.Blobs) != 1 {
-		return nil, ocispec.Descriptor{}, fmt.Errorf("signature manifest requries exactly one signature envelope blob, got %d", len(sigManifest.Blobs))
+	if len(signatureBlobs) != 1 {
+		return nil, ocispec.Descriptor{}, fmt.Errorf("signature manifest requries exactly one signature envelope blob, got %d", len(signatureBlobs))
 	}
-	sigDesc := sigManifest.Blobs[0]
+	sigDesc := signatureBlobs[0]
```

Comment:
Should this block of code in `getSignatureBlobDesc()` since `getSignatureBlobDesc()` should return just one descriptor by its method name?